### PR TITLE
Ka add schools to picker

### PR DESCRIPTION
**Pull request by kailanjian**
## Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. 

## Types of changes

- [ ] I've labeled this PR with appropriate labels (`comp/*` component, bug/feature, documentation and other less important ones).

## Checklist
_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [ ] I've linked the proper todo
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests (unit, integration...etc.) that prove my fix is effective or that my change works
- [ ] I have added or updated necessary documentation (if appropriate). You can search, for instance using the [`silver searcher`](https://github.com/ggreer/the_silver_searcher) `ag <STRING>` to find all usages in the repository.
- [ ] I have notified the team of any breaking or important changes

### CODE REVIEW

File: frontend/src/pages/ApplicationForm/ApplicationForm.jsx [hunk 1]
```diff
@@ -203,7 +203,7 @@ const ApplicationFormValidator = (handleFeedbackChange, inputs, t) => {
     },
     validateTermsChecked: (fieldName, value) => {
       if (value !== 'checked') {
-        handleFeedbackChange(fieldName, 'validations.check_terms');
+        handlppliceFeedbackChange(fieldName, 'validations.check_terms');
```
Comment (richyen) [inline frontend/src/pages/ApplicationForm/ApplicationForm.jsx:206]: is this a typo?

**Review by richyen — CHANGES_REQUESTED** [review]
otherwise looks ok

File: frontend/src/utils/data_files/sorted_schools_list.js [hunk 2]
```diff
@@ -0,0 +1,739 @@
+
```
Comment (richyen) [inline frontend/src/utils/data_files/sorted_schools_list.js:1]: For the record, if we ever want *all* schools, I think we'd pull from here:
https://ope.ed.gov/dapip/#/download-data-files

File: frontend/src/pages/ApplicationForm/ApplicationForm.jsx (hunk 1 above)
Comment (kailanjian) [reply frontend/src/pages/ApplicationForm/ApplicationForm.jsx]: @richyen this is fixed!

File: frontend/src/utils/data_files/sorted_schools_list.js (hunk 2 above)
Comment (kailanjian) [reply frontend/src/utils/data_files/sorted_schools_list.js:1]: I used Dustin's file that has mappings to regions, other schools wouldn't be auto-mapped anyway so it's ok if people type them in, but maybe we can do that as the need arises

**Pull request merged**